### ISSUE TITLE
Allow two bots with the same token to use the same slash command set

### ISSUE THREAD

**Issue by 6r1d** [interactions-py/interactions.py#354]
Hello. Recently, I noticed that if I run two bots with the same Discord token, registering the simple coroutine to handle the slash command, new bot will overwrite slash commands for the old bot.

The new bot instance one will function correctly, but older will throw an exception:

```python
    await func.invoke(ctx, **args)
  File "/home/grid/.local/lib/python3.8/site-packages/discord_slash/model.py", line 210, in invoke
    return await self.func(*args, **kwargs)
  File "./bot.py", line 583, in slash_continue
    await ctx.defer()
  File "/home/grid/.local/lib/python3.8/site-packages/discord_slash/context.py", line 153, in defer
    await self._http.post_initial_response(base, self.interaction_id, self._token)
  File "/home/grid/.local/lib/python3.8/site-packages/discord/http.py", line 250, in request
    raise NotFound(r, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
```

This repeats both with decorator registration and with adding commands manually, no `sync_all_commands` call.

```python
async def slash_whatever(ctx, input_text: str):
    await ctx.send(input_text)

self.slash.add_slash_command(slash_whatever, name="whatever", guild_ids=self.guild_ids)
```

**Context**:

```
discord-py-interactions @ git+https://github.com/goverfl0w/discord-interactions@6db2e5bdfb6b99f6081c88ab31163257601bf99b
discord.py==1.7.3
```

I wonder if it's possible to change that behavior, so both bots receive the slash command?
My goal is to have two bots to handle different channels, so I can code something while the stable version is running.
Thanks!

**P.S.**: "documentation" label was added automatically, I doubt it's needed here.

**Comment by Catalyst4222** [comment]
That happens because one bot replies, discord removes the interaction since it succeeds, then the second bot tries to respond, but the interaction doesn't exist anymore
tl;dr discord's fault

**Comment by 6r1d** [comment]
> That happens because one bot replies, discord removes the interaction since it succeeds, then the second bot tries to respond, but the interaction doesn't exist anymore
Oh, that's inconvenient.

I wonder if there's a roundabout way to handle it and any trace of the command name?
I don't even need additional arguments to handle it. As-is Discord slash commands seem worse than simply adding commands manually...

**Comment by Catalyst4222** [comment]
You can set manual checks in your code to have one copy only handle commands in certain channels

> As-is Discord slash commands seem worse than simply adding commands manually...

The discord devs are working on improving them, and a lot of good stuff is in the pipeline. Your specific issue with command "collision" isn't specific to slash commands though, it would happen with message commands too

**Comment by i0bs** [comment]
This is not an issue relevant to our library, and honestly is not one either with how slash commands are developed by Discord. It makes sense to have a collision-proof model because there is no reason for an interaction to still exist once it's already made a response. After then, you're better off manually making your `PATCH` request to edit/update the interaction response yourself.

**Comment by i0bs** [comment]
Closing in favor of #288 .
